### ISSUE TITLE
Blazored.FluentValidation is not compatible with new FluentValidation 11.x.x

### ISSUE THREAD

**Issue by Schaeri** [Blazored/FluentValidation#138]
**Is your feature request related to a problem? Please describe.**
Nuget package update of FluentValidation is blocked by Blazored.FluentValidation. 

**Describe the solution you'd like**
Update Blazored.FluentValidation with new FluentValidation package

**Comment by chrissainty** [comment]
Duplicate of #136
